ledger/participant-integration-api/src/test/suite/scala/platform/index/ContractStoreBasedMaximumLedgerTimeServiceSpec.scala

```diff
@@ -177,8 +177,7 @@ class ContractStoreBasedMaximumLedgerTimeServiceSpec extends AsyncFlatSpec with
         contractId3,
         contractId4,
       )
-    ).map { result =>
-      val MaximumLedgerTime.Archived(archivedResults) = result
+    ).collect { case MaximumLedgerTime.Archived(archivedResults) =>
```

Comment:
I am concerned here about the same problem as I suggested in the previous comment - if it is empty - it wont fail anymore

Reply:
:+1: